packages/manager/apps/telecom/src/app/telecom/pack/slots/informations/informations.html

```diff
@@ -229,6 +229,26 @@
                         </span>
                     </div>
                 </li>
+                <li class="oui-tile__item">
+                    <div class="oui-tile__definition">
+                        <strong class="oui-tile__term d-block" data-translate="pack_xdsl_information_links">
+                        </strong>
+                        <div>
+                            <a href="{{:: PackInformation.servicesStateLink.igmap }}" target="_blank" rel="noopener">
```

Comment:
```suggestion
                            <a data-ng-href="{{:: PackInformation.servicesStateLink.igmap }}" target="_blank" rel="noopener">
```
Also no need for a `.oui-link-icon` ?